freqtrade/utils.py

```diff
@@ -354,3 +356,105 @@ def start_test_pairlist(args: Dict[str, Any]) -> None:
             print(rapidjson.dumps(list(pairlist), default=str))
         else:
             print(pairlist)
+
+
+def start_hyperopt_list(args: Dict[str, Any]) -> None:
+    """
+    List hyperopt epochs previously evaluated
+    """
+    from freqtrade.optimize.hyperopt import Hyperopt
```

Comment:
Just noticed that neither of these functions are touched by tests.

We don't need full coverage - especially not for utils. 

but i think having a quick "does this work if i call it" test will be useful and is probably required - considering that hyperopt is a class with frequent changes (and it won't stop - reading your ideas - which is a good thing). 

currently, we would need to do manual tests of both of these functions whenever we touch the hyperopt class.
I'm 100% certain that we'll forget that sooner (rather than later) and these functions may end up similar to the scripts we used to have - which were broken whenever we tried to use them - which i think we should aim to avoid.